apphost/src/runtime.rs

```diff
@@ -145,6 +145,9 @@ pub fn launch_app_binary(runtime_dir: Option<&Path>, app_info: &AppInfo) -> Resu
     let args: Vec<PdCString> = env::args_os().map(PdCString::from_os_str).collect::<Result<_, _>>()?;
     let app_path = PdCString::from_os_str(app_info.app_path.as_os_str())?;
 
+    //Apply required setup or hacks
+    prelaunch_setup();
```

Comment:
Please rename to something more verbose, i.e. (also taking into account the suggestion below)
```rs
#[cfg(unix)]
unix::fixup_signal_handling();
```